Update signup.php with Toggle Show/hide password

maroufwani opened the pull request.

maroufwani committed:
Update signup.php

PlanetTheCloud commented:
Hi, thank you for the pull request, I see room for optimizations. Will take a look into it.

PlanetTheCloud committed:
Update signup.php

Changed element from label to button.
Simplified Javascript code.
Added icon change depending on the state of the input.

PlanetTheCloud commented:
Thank you for your contribution! Have a nice day.
